pkgs/applications/version-management/git-and-tools/gitin/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, pkgconfig, libgit2 }:
```

This one is up for your taste, but there's an unwritten convention agreed by some people to use:

```suggestion
{ stdenv
, buildGoPackage
, fetchFromGitHub
, pkgconfig
, libgit2
}:
```

This makes future diffs easier to read but it's not such a big deal.